spec/report_spec.rb

```diff
@@ -58,6 +58,46 @@
       expect(report).to be_an_instance_of(described_class)
       expect(report.id).to match('trkrep')
     end
+
+    it 'creates a report with custom columns and additional columns' do
+      report = described_class.create(
+        start_date: Fixture.report_start_date,
+        end_date: Fixture.report_end_date,
+        type: 'shipment',
+        columns: ['usps_zone'],
+        additional_columns: %w[from_name from_company],
```

Comment:
We don't typically use `%w` in our internal Ruby projects, let's use the more generic individual string approach instead of being clever here.

Reply:
The linter was complaining otherwise

Reply:
Ok, speaking of we should probably update the easycop rules in this project soonish as they get updated frequently.